benchexec/result.py

```diff
@@ -55,6 +57,8 @@
 _PROP_LABEL =        'unreach-label'
 # currently used by SV-COMP (http://sv-comp.sosy-lab.org/2016/rules.php):
 _PROP_CALL =         'unreach-call'
+# extends _PROP_CALL for any error function:
+_PROP_CALL_SPECIFIC ='unreach-call,'
```

Comment:
These constants are purely internal, so reorganize them just as you wish, as long as output is not affected.